code/LINDA/LINDA_fire.dm

```diff
@@ -26,13 +26,23 @@
 	if((exposed_temperature > PLASMA_MINIMUM_BURN_TEMPERATURE) && air_contents.toxins > 0.5)
 		igniting = 1
 
+	var/obj/effect/decal/cleanable/liquid_fuel/liquid = locate() in src
```

Comment:
@ZomgPonies Plasma only spreads at fairly high quantities. I'd have to go digging to find the exact value at which it decides to spread.

And this is LINDA code, it handles hotspot creation on turfs.